presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -382,3 +382,119 @@ node.
 
 You should ensure that the keytab files have the correct permissions on every
 node after distributing them.
+
+
+Hive Data Stored in Amazon S3
+=============================
+
+The Hive Connector can read data from files stored in S3, assuming the Hive
+Metastore has tables defined as "external" which have locations defined as
+S3 URIs.
+
+Presto registers its own S3 filesystem for the following URI schemes:
+``s3://``, ``s3a://``, ``s3n://`` and it registers the default Hadoop S3
+FileSystem (``org.apache.hadoop.fs.s3.S3FileSystem``) for ``s3bfs://`` URIs.
+The following assumes you are using the native Presto S3 FileSystem implementation.
+
+Configuration Properties
```

Comment:
Some of the properties here and most of the properties below in the tuning section are not about security, so I am not sure whether we should put all these in the `hive-security` documentation. Maybe we should add these to the hive connector documentation, what do you think @electrum?